admin_manual/appliance/Active_Directory.rst

```diff
@@ -0,0 +1,13 @@
+================
+Active Directory
+================
+
+You have tested the appliance with your Active Directory environment, 
+
+deleted it and now want to go in to production or test it again and 
+
+you can't include the appliance in your environment again?
+
+The solution is - you have to clean up the prevouis DNS entries on your Domain Controller.
```

Comment:
s/prevouis/previous/g
s/on your/in your/g
s/The solution is - you have to clean up/The solution is to clean up/g